content/authors/emmanuel-alege/index.md

```diff
@@ -0,0 +1,4 @@
+[Emmanuel Alege](https://www.section.io/engineering-education/authors/emmanuel-alege/)
+
```

Comment:
Follow other samples in the content/authors folder to format this file

Reply:
done